pandas/tests/frame/test_merge.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import print_function
```

Comment:
I see you already added to the correct location. no point in duplicating tests. delete these.